pkg/sosreport/manifest.json.in

```diff
@@ -6,7 +6,7 @@
 
     "tools": {
         "index": {
-            "label": "Diagnostic Reports",
+            "label": "Diagnostic reports",
```

Comment:
If we adopt the navigation as title case PF4 recommendation, then this should remain Title Case.